client/packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -152,9 +152,15 @@ export const AutocompleteList = <T,>({
             },
           },
           listbox: {
-            style: {
+            sx: {
               minHeight: height ? `${height}` : 'auto',
               maxHeight: height ? `${height}` : 'auto',
+              '& .MuiTypography-root': {
+                maxWidth: 497,
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+              },
```

Comment:
I'm actually very hesitant to approve something like this because Autocomplete is used in so many places, and I don't fully understand the ramifications of these changes.

Particularly unsure about the hard coded values. I wonder would this affect other places Autocomplete is used?

Reply:
This would affect other places for Autocomplete. Though shouldn't be harmful bur rather helpful.